src/metrics.js

```diff
@@ -12,8 +14,21 @@ function Metrics({ reporters, errback }) {
   }
 
   function space(key, tags) {
+    validate({
+      name: 'tags', value: tags, type: 'object', required: false,
+    });
+
+    let allTags;
+
+    if (tags || defaultTags) {
+      allTags = {
+        ...defaultTags,
+        ...tags,
```

Comment:
if one of them is not defined it will still work ?

Reply:
indeed. There are tests for that scenario